kotlin-android/app/build.gradle

```
@@ -30,7 +30,7 @@ android {
 
     buildTypes {
         def mongodbRealmUrl = "https://realm-dev.mongodb.com"
-        def appId = "<your app ID here>"  // Replace with proper Application ID
+        def appId = "myrealmapp-vjmee" //"<your app ID here>"  // Replace with proper Application ID
```

I don't think you meant to commit this.
```suggestion
        def appId = "<your app ID here>"  // Replace with proper Application ID
```